packages/framework/src/Model/Customer/User/CustomerUserRepository.php

```diff
@@ -196,4 +196,23 @@ public function getOneByUuid(string $uuid): CustomerUser
 
         return $customerUser;
     }
+
+    /**
+     * @param int $salesRepresentativeId
+     * @return array
+     */
+    public function findCustomerUsersUsingSalesRepresentative(int $salesRepresentativeId): array
+    {
+        $customers = $this->getCustomerUserRepository()
+            ->createQueryBuilder('c')
+            ->select('c')
+            ->where('c.salesRepresentative = :salesRepresentativeId')
+            ->setParameter('salesRepresentativeId', $salesRepresentativeId)
+            ->getQuery()
+            ->getArrayResult();
```

Comment:
I believe this can be simplified to the following:
```php
        $customers = $this->getCustomerUserRepository()->findBy([
            'salesRepresentative' => $salesRepresentativeId,
        ]);
```

Reply:
you are right